lib/sisimai/bite/email/exchange2007.rb

```diff
@@ -133,8 +133,8 @@ def scan(mhead, mbody)
               else
                 if v['diagnosis'].to_s.size > 0 && v['diagnosis'].end_with?('=')
                   # Continued line of error messages
-                  v['diagnosis']  = v['diagnosis'].sub(/=\z/, '')
-                  v['diagnosis'] += e
+                  v['diagnosis']  = v['diagnosis'].chomp('=')
```

Comment:
__[RuboCop]__

Layout/SpaceAroundOperators: Operator `=` should be surrounded by a single space.


<!-- Comment by [SideCI](https://sideci.com) -->